qiskit/qasm3/exporter.py

```diff
@@ -465,7 +465,8 @@ def build_subroutine_definition(self, instruction):
             # the definition.
             raise QASM3ExporterError(
                 "Exporting subroutines with parameters is not yet supported by the OpenQASM 3"
-                " exporter. Received this instruction, which appears parameterized:\n{instruction}"
+                " exporter. Received this instruction, which appears parameterized:"
+                f"\n{instruction}"
```

Comment:
I think you probably could have left this as 1 line just with `f`, but it doesn't really matter.